### PR TITLE
add pymumble_py3k for python 3.* support

### PR DESCRIPTION
**Pull request by ranomier**
Just changes things needed for python 3.*

### CODE REVIEW

File: pymumble_py3k/tools.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+import struct
+import builtins
```
Comment (Lartza) [inline pymumble_py3k/tools.py:3]: As far as I can see, builtins is imported here unnecessarily?

**Review by Lartza — CHANGES_REQUESTED** [review]
Almost everything seems to be in good shape, testing this fully requires a bit more time.

Protobuf 3.0.0 should be used for Python 3 and mumble_pb2.py regenerated with that.

File: pymumble_py3k/tools.py (hunk 1 above)
Comment (ranomier) [reply pymumble_py3k/tools.py:3]: Yes you are right, sry forgot :(

**Comment by ranomier** [comment]
Closing wrong branch

**Pull request closed without merging**